package/INI/MPMaps.ini

```diff
@@ -8179,4 +8164,84 @@ Waypoint3=128180
 Waypoint4=178130
 Waypoint5=177106
 Waypoint6=127057
-Waypoint7=104056
\ No newline at end of file
+Waypoint7=104056
+
+[Maps\Yuri's Revenge\2_Indonesia_navalwar]
+Author=Westwood & Instinct Edit
```

Comment:
Indonesia is westwood?

Reply:
There was no author in the map and I found the name of this author in the Chinese maps channel on discord.